### PR TITLE
feat: remove hello-world service (#200)

### PR DESCRIPTION
**Pull request by lasiva**

### CODE REVIEW

File: pre_install_report/library/pre_install_check.py [hunk 1]
```diff
@@ -197,30 +195,30 @@ def check_details(self, kubectl, ingress_port, ingress_host, ingress_controller,
...
-        params[viya_constants.INGRESS_HOST] = str(ingress_host)
-        params[viya_constants.INGRESS_PORT] = str(ingress_port)
         params[viya_constants.PERM_CLASS] = utils
         params[viya_constants.SERVER_K8S_VERSION] = self._k8s_server_version
         params['logger'] = self.sas_logger
 
         permissions_check = PreCheckPermissions(params)
         self._check_permissions(permissions_check)
 
-        test_list = [viya_constants.INSUFFICIENT_PERMS, viya_constants.PERM_SKIPPING]
```
Comment (kevinlinglesas) [inline pre_install_report/library/pre_install_check.py:210]: Can you elaborate on why this split of test_list to two different results processed separately going forward.  Comments would be helpful here.

File: pre_install_report/library/pre_install_check_permissions.py [hunk 2]
```diff
@@ -625,7 +333,7 @@ def check_deploy_crd(self):
     def check_rbac_role(self):
         """
         Check if RBAC is enabled in specified namespace.
-        Create the Role, Rolebinding, and Serviec account in the specified namespace. Set the
+        Create the Role, Rolebinding, and Service account in the specified namespace. Set the
```
Comment (kevinlinglesas) [inline pre_install_report/library/pre_install_check_permissions.py:336]: Luv finding these little nuggets ;)

File: pre_install_report/test/test_data/json_data/azure_nodes_no_master.json [hunk 3]
```diff
@@ -1639,7 +1639,7 @@
                     },
                     {
                         "names": [
-                            "gcr.io/google-samples/node-hellofhds058d238d0ab54efb76ec0f7b1da666cefa9b40be59ef34346a761b8adc2dd45459b"
+                            "cr.io/my-samples/node-hellofhds058d238d0ab54efb76ec0f7b1da666cefa9b40be59ef34346a761b8adc2dd45459b"
```
Comment (kevinlinglesas) [inline pre_install_report/test/test_data/json_data/azure_nodes_no_master.json:1642]: Why do we retain any reference to a node-hello image?   Dup this comment a few times more in the changeset.

**Review by kevinlinglesas — COMMENTED** [review]
Had a few comments/questions.

File: pre_install_report/library/pre_install_check.py (hunk 1 above)
Comment (kevinlinglesas) [reply pre_install_report/library/pre_install_check.py]: Discussed this with @lasiva.   The approach initially taken to split test_list into two different results was re-considered and initial changes reverted.    The minor issue warning message will be addressed separately now.

File: pre_install_report/test/test_data/json_data/azure_nodes_no_master.json (hunk 3 above)
Comment (kevinlinglesas) [reply pre_install_report/test/test_data/json_data/azure_nodes_no_master.json]: The node-hello reference in pytest data files was unrelated to our removal of the functionality that actually uses that image.   That image was chosen likely as a conveniently known image name to stick in test json file.   Different references now.

**Review by kevinlinglesas — APPROVED** [review]
lgtm

**Review by lasiva — COMMENTED** [review]
removed logger message  self.logger.warn("WARN: Review Namespace Permissions") in pre_install_checkp.py

**Review by kevinlinglesas — APPROVED** [review]
latest changes lgtm.

**Pull request merged**